src/components/BrowsePage.js

```diff
@@ -30,69 +36,62 @@ function BrowsePage({
       </div>);
   }
 
-  if (isLoading) {
-    return (
-      <section className="container">
-        {titleTabs}
-        loading...
-      </section>
-    );
-  }
-
   if (error) {
-    return (
-      <section className="container">
-        {titleTabs}
-        <ErrorMessage>Oops! Could not load homepage :(</ErrorMessage>
-      </section>
+    statusMsg = (
+      <ErrorMessage>{error}</ErrorMessage>
     );
-  }
-
-  if (type === "movies") {
-    return (
-      <section className="container">
-        {titleTabs}
-        <Switch>
-          <Route
-            exact
-            path="/movies/genre/"
-            render={props =>
-              <BrowseGenresContainer {...props} genres={genres} type={type} />
-            }
-          />
-          <Route
-            path="/movies/:filter"
-            render={() => (
-              <PosterGrid movies={movies} />
-            )}
-          />
-        </Switch>
-      </section>
+  } else if (isLoading) {
+    statusMsg = (
+      <div>
+        loading...
```

Comment:
Willy likes the loading text with capital L, else he will be annoyed about it for several weeks =(

Reply:
Ouch :( I can't hurt his feelings so I will have to change it.

Reply:
Lmao you guys